fe/fe-core/src/main/java/com/starrocks/sql/optimizer/task/ApplyRuleTask.java

```diff
@@ -115,12 +115,15 @@ public void execute() {
             }
 
             GroupExpression newGroupExpression = result.second;
-            if (sessionVariable.isEnableMaterializedViewForceRewrite()) {
```

Comment:
why ?

Reply:
`appliedRules ` is used in `force rewrite` before, but we can use it for `Binder` too since it's trivial to record which rules are applied?